Makefile.toml

```
@@ -259,11 +260,19 @@ chmod -R o+r ${CARGO_HOME}
 dependencies = ["fetch-sdk"]
 script = [
 '''
-${BUILDSYS_TOOLS_DIR}/docker-go \
-  --module-path ${BUILDSYS_SOURCES_DIR}/host-ctr \
-  --sdk-image ${BUILDSYS_SDK_IMAGE} \
-  --go-mod-cache ${GO_MOD_CACHE} \
-  --command "go list -mod=readonly ./... >/dev/null && go mod vendor"
+function go_fetch
+{
+  local module=$1
+  ${BUILDSYS_TOOLS_DIR}/docker-go \
+    --module-path ${BUILDSYS_SOURCES_DIR}/$module \
```

pedantic style nit:
```suggestion
    --module-path "${BUILDSYS_SOURCES_DIR}/${module}" \
```

I'd recommend just always using braces and quotes for variables to avoid needing to think about when they're optional or might be required.